k8s/e2e/common/test_common.go

```diff
@@ -179,8 +179,12 @@ func Test5NodesSoakEnvironmentWithPVCs(t *testing.T) {
 	})
 }
 
-func TestWithSingleNodeEnv(t *testing.T) {
+func TestWithSingleNodeEnvParallel(t *testing.T) {
```

Comment:
I'm just curious what's the point of having a separate one for this?

Reply:
I need to set an environment variable in the ones that tests with local qa-charts. You can't set environment variables in parallel tests since they will affect other tests running in parallel.